test/amplitude-client.js

```diff
@@ -54,6 +54,17 @@ describe('AmplitudeClient', function() {
       assert.equal(new AmplitudeClient('$DEFAULT_INSTANCE')._instanceName, '$default_instance');
     });
 
+    it('should invoke onInit callbacks', function() {
+      let onInitCalled = false;
+      let onInit2Called = false;
+      amplitude.onInit(() => { onInitCalled = true; });
```

Comment:
Another test you might want to do (not sure if you can automate it, might have to manually do it), is verify that the functions are being queued even when the SDK hasn't loaded. So like maybe mess with the src of the snippet so that the SDK doesn't load, and verify that the dud queue still has onInit callbacks.

Reply:
Actually we don't need to do anything too complicated. Code within the snippet script tag will always load before the SDK loads from the snippet.

Tested. Looks like it works as hoped.